charts/redpanda/go.mod

```diff
@@ -0,0 +1,219 @@
+module github.com/redpanda-data/redpanda-operator/charts/redpanda
+
+go 1.23.5
+
+require (
+	github.com/cert-manager/cert-manager v1.16.3
+	github.com/cockroachdb/errors v1.11.3
+	github.com/invopop/jsonschema v0.13.0
+	github.com/json-iterator/go v1.1.12
+	github.com/pkg/errors v0.9.1
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.79.2
+	github.com/quasilyte/go-ruleguard/dsl v0.3.22
+	github.com/redpanda-data/common-go/rpadmin v0.1.12
+	github.com/redpanda-data/redpanda-operator/charts/connectors v0.0.0-20250123130424-d2562617eac6
+	github.com/redpanda-data/redpanda-operator/charts/console v0.0.0-20250123130424-d2562617eac6
```

Comment:
Hm, so this is going to be interesting. IMO this dep is technically invalid because the commit isn't present in `main` and the commit will change once this is rebased into `main`. The linter PR should reject this.

I think the temporary solution here would be adding a replace directive until this is merged into main at which point we can pin to the minted commit.

And according to the [modules wiki](https://go.dev/wiki/Modules#is-it-possible-to-add-a-module-to-a-multi-module-repository) we'll want to add a blank dependency on the `charts` module. Have you given that a shot?

Reply:
I pushed tags to this repo to correctly reference dependency. If I don't remove branch then it will work until I bump this dependency.

Reply:
My preference would be to use `replace` in situations like this so we can guarantee that commits in `main` will be evergreen and not reliant on an arbitrary commit object they may or may not be GC'd. Is there any reason to not (temporarily) use replace in this instance?